### DISCUSSION TITLE
Change network access address

### DISCUSSION THREAD

**Discussion by codyspb** [vladmandic/sdnext#2295] [Q&A] [answered]
Good afternoon.
Please tell me, is it possible to access webui from another computer?
I tried to open the address using real IP:port, but the page did not open. Is it possible to indicate somewhere which address the project should listen to?

**Comment by vladmandic** [comment, marked as answer]
did you start with `webui --listen`? if not, it only uses loopback (only available on same computer).

**Reply by codyspb** [reply]
Thx! It's works!
